src/almanack/LoC_tracker.py

```diff
@@ -0,0 +1,26 @@
+"""
+This module calculates the absolute value of lines of code (LoC) changed (added or removed)
+in the given Git repository.
+"""
+
+import git
+
+
+def calculate_loc_changes(repo_path: str):
+    """
+    Finds the total number of code lines changed
+
+    Args:
+        repo_path(str): The path to the git repository
+    Returns:
+        int: Total number of lines added or removed
+    """
+    repo = git.Repo(repo_path)
+
+    total_lines_changed = 0
+
+    for commit in repo.iter_commits():
```

Comment:
Would it be possible to tie in `almanack.git_parser.get_commit_logs` here somehow to help introduce flexibility when it comes to certain commit segments or time periods (instead of only in full summary)? Perhaps it could come in the form of an expansion of the data structure found within that function (another dictionary key-value pair, for example). For example, we might expect that earlier in a project's lifespan the amount of change will be higher than later on. These counts from the earlier time periods might inadvertently effect calculations for later time periods under certain circumstances.

Reply:
Thanks for the idea Dave! I was able to implement the get_commit_logs by adding in attributes the my function, but wanted to see what you think?